crates/bevy_reflect/src/impls/std.rs

```diff
@@ -41,6 +41,11 @@ impl_reflect_value!(String(Debug, Hash, PartialEq, Serialize, Deserialize));
 impl_reflect_value!(Result<T: Clone + Reflect + 'static, E: Clone + Reflect + 'static>());
 impl_reflect_value!(HashSet<T: Hash + Eq + Clone + Send + Sync + 'static>());
 impl_reflect_value!(Range<T: Clone +  Send + Sync + 'static>());
+impl_reflect_value!(RangeInclusive<T: Clone +  Send + Sync + 'static>());
```

Comment:
Nit: Not that you _have_ to, since none of the others do this, but you could replace some of those bounds with `Reflect` itself:
```suggestion
impl_reflect_value!(RangeInclusive<T: Clone + Reflect>());
```